pkg/helm/option.go

```diff
@@ -53,7 +53,7 @@ type options struct {
 	disableHooks bool
 	// if set, skip CRD hook only
 	disableCRDHook bool
-	// name of release
+	// if set, render SubChart Notes
```

Comment:
did you mean to make this comment change?

Reply:
Oops!  No, there had been a var there before that was removed.  I'll revert that line.